Mathlib/Geometry/Euclidean/Angle/Unoriented/TriangleInequality.lean

```diff
@@ -0,0 +1,220 @@
+/-
+Copyright (c) 2025 Ilmārs Cīrulis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ilmārs Cīrulis
+-/
+import Mathlib.Geometry.Euclidean.Angle.Unoriented.Affine
+
+/-!
+# The Triangle Inequality for Angles
+
+This file contains proof that angles obey the triangle inequality.
+-/
+
+open InnerProductGeometry
+
+open scoped RealInnerProductSpace
+
+variable {V : Type*}
+variable [NormedAddCommGroup V]
+variable [InnerProductSpace ℝ V]
+
+/-- **Gets unit length vector from given vector** -/
+noncomputable def unit (x : V): V := ‖x‖⁻¹ • x
+
+lemma norm_of_unit {x : V} (H : x ≠ 0) : ‖unit x‖ = 1 := by
+  unfold unit; rw [norm_smul];
+  simp only [norm_inv, norm_norm]
+  have H0 : ‖x‖ ≠ 0 := by simp only [ne_eq, norm_eq_zero]; exact H
+  field_simp
+
+lemma inner_product_of_units_as_cos {x y : V} (Hx : ‖x‖ = 1) (Hy : ‖y‖ = 1) :
+    ⟪x, y⟫ = Real.cos (angle x y) := by
```

Comment:
The name by the naming conventions should be something like `inner_eq_cos_angle_of_norm_eq_one`. Similarly, please review other names against the naming conventions.